features/support/helpers.rb

```diff
@@ -34,13 +34,14 @@ def sign_in_to_front_end_if_necessary(email)
   )
 end
 
-def sign_in_to_back_office(user)
+def sign_in_to_back_office(user, force = true)
```

Comment:
I have added this `force = true`. This tells the function to quit if ANY user is signed in. I have decided to do that because observing the `convictions.feature` we have a very clear 
```
  Background:
   Given an Environment Agency user has signed in to the backend
```
but then this function is called and re-sign in the user to be an arbitrary one. So, to avoid unnecessary logout / login I have decided to go for this. Let me know what you think.